content/tutorial-deep-reinforcement-learning-with-pong-from-pixels.ipynb

```diff
@@ -0,0 +1,1275 @@
+{
```

Comment:
<p>It might be worth mentioning that this command applies to people using venv, but not conda. Also, I'm not sure we should be installing packaged using a jupyter notebook. This may create issues for users in Windows, for example. I'd prefer something like:</p><p> </p><p><code>First, you should install OpenAI Gym using pip or conda, and import NumPy, Gym and the necessary modules:</code></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/numpy/numpy-tutorials/pull/35/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
Good idea. Done 👍

Reply:
The word "First" is repeated, PiPy is a mistake I think (you don't need to mention it - saying it can be installed using pip is quite enough)